HOW_WE_USE_GITHUB.md

```diff
@@ -226,30 +261,59 @@ please post details to the [Nucleus forums](https://community.anaconda.cloud/).
 </pre>
 
 > **Warning**
-> Apply the https://github.com/conda/infra/labels/off-topic label to these tickets before closing them out.
+> Apply the https://github.com/conda/infrastructure/labels/off-topic label to these issues before closing them out.
 
 </details>
 
-In order to not have to manually type or copy/paste the above repeatedly, please note that it's possible to add text for the most commonly-used responses via [GitHub's "Add Saved Reply" option][docs-saved-reply].
-
+In order to not have to manually type or copy/paste the above repeatedly, note that it's possible to add text for the most commonly-used responses via [GitHub's "Add Saved Reply" option][docs-saved-reply].
 
-### Commit signing
+## Commit Signing
 
 For all conda maintainers, we require commit signing and strongly recommend it for all others wishing to contribute to conda
 related projects. More information about how to set this up within GitHub can be found here:
 
-- [Signing Commits][signing-commits]
+- [GitHub's signing commits docs][docs-commit-signing]
 
-### Types of Tickets
+## Types of Issues
 
-#### Standard Ticket/Issue
+### Standard Issue
 
 TODO
 
-#### Epics
+### Epics
 
 TODO
 
-#### Spikes
+### Spikes
 
-TODO
+#### What is a spike?
+
+"Spike" is a term that is borrowed from extreme programming and agile development. They are used when the **outcome of an issue is unknown or even optional**. For example, when first coming across a problem that has not been solved before, a project may choose to either research the problem or create a prototype in order to better understand it.
+
+Additionally, spikes represent work that **may or may not actually be completed or implemented**. An example of this are prototypes created to explore possible solutions. Not all prototypes are implemented and the purpose of creating a prototype is often to explore the problem space more. For research-oriented tasks, the end result of this research may be that a feature request simply is not viable at the moment and would result in putting a stop to that work.
+
+Finally, spikes are usually **timeboxed** however given the open source/volunteer nature of our contributions we do not enforce this on our contributors. When a timebox is set, this means that we are limiting how long we want someone to work on said spike. We do this to prevent contributors from falling into a rabbit hole they may never return from. Instead, we set a time limit to perform work on the spike and then have the assignee report back. If the tasks defined in the spike have not yet been completed, a decision is made on whether it makes sense to perform further work on the spike.
```

Comment:
```suggestion
Finally, spikes are usually **timeboxed**. However, given the open source/volunteer nature of our contributions, we do not enforce this for our contributors. When a timebox is set, this means that we are limiting how long we want someone to work on said spike. We do this to prevent contributors from falling into a rabbit hole they may never return from. Instead, we set a time limit to perform work on the spike and then have the assignee report back. If the tasks defined in the spike have not yet been completed, a decision is made on whether it makes sense to perform further work on the spike.
```